### DISCUSSION TITLE
Azure AD [Jun 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2570] [Discord Threads]
> 🙋‍♂️ **Trung BUI**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### unfortunately, not available at the moment  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/990960220681408533)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
